Vagrantfile

```diff
@@ -29,7 +37,7 @@ Vagrant.configure(2) do |config|
 SCRIPT
 
   config.vm.provision "ansible" do |ansible|
-    ansible.playbook = "deployment/ansible/pc-demo.yml"
+    ansible.playbook = "deployment/ansible/rastervision.yml"
```

Comment:
I don't know enough to know whether this rename is accurate or not. Are you sure it makes sense to name things here after Raster Vision generically?

Reply:
This app serves data generated by raster-vision, but is not raster-visions itself, so perhaps isprs-potsdam-viz is a better name. I think raster-vision should just be used to refer to the raster-vision repo.